pkg/controller/operators/olm/operator.go

```diff
@@ -2087,7 +2097,15 @@ func (a *Operator) transitionCSVState(in v1alpha1.ClusterServiceVersion) (out *v
 		}
 
 		// If there is a succeeded replacement, mark this for deletion
-		if next := a.isBeingReplaced(out, a.csvSet(out.GetNamespace(), v1alpha1.CSVPhaseAny)); next != nil {
+		next := a.isBeingReplaced(out, a.csvSet(out.GetNamespace(), v1alpha1.CSVPhaseAny))
```

Comment:
This block of code is to determine the oldest CSV in the CSV replacing chain so it can be deleted first. Please put a comment here to reflect that. It helps with the review process and future changes.